html/dom/aria-attribute-reflection.tentative.html

```diff
@@ -57,4 +59,531 @@
     }, "aria-rowindextext attribute reflects.");
 </script>
 
+<!-- tentative: https://github.com/w3c/aria/pull/2484 -->
+<!-- ARIA attributes to be converted to enumerated:
+    - aria-atomic
+    - aria-autocomplete
+    - aria-busy
+    - aria-checked
+    - aria-current
+    - aria-disabled
+    - aria-expanded
+    - aria-haspopup
+    - aria-hidden
+    - aria-invalid
+    - aria-live
+    - aria-modal
+    - aria-multiline
+    - aria-multiselectable
+    - aria-orientation
+    - aria-pressed
+    - aria-readonly
+    - aria-required
+    - aria-selected
+    - aria-sort
+-->
+<!-- aria-atomic -->
+<div id="aria_atomic"></div>
+<script>
+test(function(t) {
+    let el = document.getElementById("aria_atomic");
+    assert_equals(el.ariaAtomic, null);
+
+    el.ariaAtomic = "true";
+    assert_equals(el.ariaAtomic, "true");
+    el.ariaAtomic = "tRuE";
+    assert_equals(el.ariaAtomic, "true");
+    el.ariaAtomic = "   true";
+    assert_equals(el.ariaAtomic, "true");
+    el.ariaAtomic = "false";
+    assert_equals(el.ariaAtomic, "false");
+    el.ariaAtomic = "";
+    assert_equals(el.ariaAtomic, "false");
+
+    el.ariaAtomic = "$%1FoOBaR";
+    assert_equals(el.ariaAtomic, "false");
```

Comment:
Presumably `true` with leading spaces and `$%1FoOBaR` are equally invalid. How come they reflect as different states?